sample-queries/sample-queries.json

```diff
@@ -1922,62 +1922,62 @@
         "skipTest": false
     },
     {
-        "id": "31464243-ffe3-4d18-82df-af05d3e45221",
+        "id": "ced9c5ee-7745-49b2-9e08-018e22dbc41a",
         "category": "Microsoft To Do",
         "method": "GET",
         "humanName": "get To Do task lists",
-        "requestUrl": "/beta/me/todo/lists",
-        "docLink": "https://docs.microsoft.com/en-us/graph/api/todo-list-lists?view=graph-rest-beta",
+        "requestUrl": "/v1.0/me/todo/lists",
+        "docLink": "https://docs.microsoft.com/en-us/graph/api/todo-list-lists?view=graph-rest-1.0",
```

Comment:
Correct the url to documentation, this url is incorrect and the right url should be "https://docs.microsoft.com/en-us/graph/api/todo-list-lists?view=graph-rest-beta&viewFallbackFrom=graph-rest-1.0&tabs=http" please correct all urls for the other samples as well.

Reply:
We are awaiting checking in of the documentation changes. Post that, the docLinks will be correct

Reply:
@bettirosengugi the documentation changes have been merged and the links are now functional